### ISSUE TITLE
NTPSync always times out when using Tor (Orbot)

### ISSUE THREAD

**Issue by bugabinga** [dschuermann/ntp-sync#19]
The "Query" ,"Detailed Query" and "Query and set time" functions always show a "Server timeout!" flash message while Tor is active for all connections using Orbot.

As a side note: Another of Dominiks apps, that uses NTP (FasterGPS) seems to be working fine under the same scenario.

Best regards, Oliver

**Comment by robinpaulson** [comment]
Yep, I get this too - I can't see anything reported in the logs according to aLogcat. It seems the author is not supporting the software any more though; I wonder if anyone else here is interested in being the lead dev and forking this repo?
